src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -4808,6 +4941,57 @@ namespace HeatBalanceManager {
 			PerformSolarCalculations();
 		}
 
+		// Initialize zone outdoor environmental variables
+		// Bulk Initialization for Temperatures & WindSpeed
+		// using the zone, modify the zone  Dry/Wet BulbTemps
+		SetZoneOutBulbTempAt();
+		CheckZoneOutBulbTempAt();
+
+		// X Luo Added 07/19/2017
+		// set zone level wind dir to global value
+		SetZoneWindSpeedAt();
+		SetZoneWindDirAt();
+
+		// X Luo Added 07/30/2017
+		// Set zone data to linked air node value if defined.
+		GetOutAirNodesInput();
```

Comment:
This should be SetOutAirNodes(), if I am following correctly.  You need to set the values here.  This also makes sure the input processing is only done once.  That should resolve the bnd diffs.